odpf/optimus/runtime_service.proto

```diff
@@ -187,6 +187,11 @@ service RuntimeService {
             post: "/v1/project/{project_name}/job/{job_name}/replay"
         };
     }
+    rpc GetReplayStatus(ReplayStatusRequest) returns (ReplayStatusResponse) {
```

Comment:
according to proto guides, each message in rpc should be prefixed by its method name so this should be 
```
GetReplayStatusRequest/GetReplayStatusResponse
```

Reply:
updated